fetch-release.sh

```
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
```

or else we just fail silently